desktop/core/src/desktop/js/reactComponents/MetricsComponent/MetricsComponent.scss

```diff
@@ -0,0 +1,48 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../components/styles/variables';
+
+.metrics-component {
+  background-color: $fluidx-gray-100;
+  padding: 0 24px 24px 24px;
+
+  .metrics-heading {
+    color: $fluidx-gray-700;
+    padding-left: $font-size-lg;
+    font-size: $fluidx-heading-h4-size;
+    line-height: $fluidx-heading-h4-line-height;
+    font-weight: $fluidx-heading-h4-weight;
+  }
+
+  .metrics-filter {
+    margin: $font-size-sm;
+    width: 30%;
+    box-shadow: none !important;
```

Comment:
I have removed the box shadow from the input filter metrics ( ! important).
Earlier it showed a blue outline as soon as u click the input filter.

Reply:
nit: try not to use !important, instead increase the specificity of the selector if required.